vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -178,5 +189,16 @@ def parse_base_item(item)
         :resource_version   => item.metadata.resourceVersion
       }
     end
+
+    HUMAN_SIZE_SUFFIXES = %w(Ki Mi Gi Ti)
+
+    def human_to_value(value)
+      exp_index = HUMAN_SIZE_SUFFIXES.index(value[-2..-1])
+      if exp_index.nil?
+        return Integer(value)
+      else
+        return Integer(value[0..-3]) * 1024**(exp_index + 1)
```

Comment:
> @Fryguy, I think the problem is that kubernetes API is returning numbers with obscure (or incorrect?) abbreviations for number groupings. E.g., "Ki" instead of "KB", "Mi" instead of "MB", "Ti" instead of "TB".

yes,
example output from k8s REST api:
"capacity": {
          "cpu": "8",
          "memory": "16341880Ki"
        }